### DISCUSSION TITLE
Change connected DB configuration params [Jul 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2762] [Discord Threads]
> 🙋‍♂️ **overedu**  
## Here is a screenshot!  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/996411219441360896/Screen_Shot_2022-07-12_at_9.41.41_AM.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@380063457099382785> Sorry! You will have to create a new project for now. Will raise a ticket to support this going forward.  
#  
> 🙋‍♂️ **overedu**  
#### That's a bummer. Thanks anyway!  
#  
> 👨‍🦲 **dstala**  
#### https://github.com/nocodb/nocodb/issues/2657  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/996411220548653096)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
